core/src/env/test_env.rs

```diff
@@ -567,6 +601,34 @@ where
     fn dispatch_raw_call(data: &[u8]) {
         TEST_ENV_DATA.with(|test_env| test_env.borrow_mut().dispatch_call(data))
     }
+
+    fn call_invoke(
+        callee: T::AccountId,
+        gas: u64,
+        value: T::Balance,
+        input_data: &[u8],
+    ) -> Result<(), CallError> {
+        let callee = &(callee.encode())[..];
+        let value = &(value.encode())[..];
+        let res = TEST_ENV_DATA
+            .with(|test_env| test_env.borrow_mut().call(callee, gas, value, input_data));
+        match res {
```

Comment:
you can replace this `match` expression with a simple combinator: `.map(|_res| ())`